File: app/src/main/java/com/theone/simpleshare/ui/pairing/PairingViewModel.java
package com.theone.simpleshare.ui.pairing;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import java.util.ArrayList;

public class PairingViewModel extends ViewModel {

    private final MutableLiveData<String> mText;
    private final MutableLiveData<ArrayList<Item>> mItemList;
    public PairingViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is pairing fragment");

        mItemList = new MutableLiveData<>();
        mItemList.setValue(new ArrayList<>());
    }

    public LiveData<String> getText() {
        return mText;
    }
    public LiveData<ArrayList<Item>> getList(){
        return mItemList;
    }
}

File: app/src/main/kotlin/com/theone/simpleshare/ui/pairing/PairingViewModel.kt
package com.theone.simpleshare.ui.pairing

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import java.util.ArrayList

class PairingViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    private val mItemList: MutableLiveData<ArrayList<Item>>
    val text: LiveData<String>
        get() = mText
    val list: LiveData<ArrayList<Item>>
        get() = mItemList

    init {
        mText = MutableLiveData()
        mText.value = "This is pairing fragment"
        mItemList = MutableLiveData()
        mItemList.value = ArrayList<Item>()
    }
}